G1ANT.Addon.Images/Extensions/ImageExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿using AForge.Imaging;
+using G1ANT.Language;
+using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Linq;
+using Image = AForge.Imaging.Image;
+
+namespace G1ANT.Addon.Images
+{
+    public static class ImageExtensions
+    {
+        public static Bitmap OpenFile(this TextStructure filePath)
+        {
+            try
+            {
+                using (var bitmap = new Bitmap(filePath.Value))
+                {
+                    return Image.Clone(bitmap, PixelFormat.Format24bppRgb);
+                }
+            }
+            catch (Exception ex)
+            {
+                throw new Exception($"Could not open the image file '{filePath.Value}'. Message: {ex.Message}", ex);
+            }
+        }
+
+        public static Bitmap OpenFileOrGetScreenshot(this TextStructure filePath, RectangleStructure screenArea, BooleanStructure relative)
```

Comment:
I would be more keen on removing "OpenImage" part of this method as it is already implemented as extension. You should decide wether to take screenshot or open image beforehand.